### PR TITLE
Replace dd calls with unix advice calls in ledmanager

### PR DESCRIPTION
**Pull request by adarsh-zededa**
Signed-off-by: adarsh-zededa <adarsh@zededa.com>

### CODE REVIEW

File: pkg/pillar/cmd/ledmanager/ledmanager.go [hunk 1]
```diff
@@ -451,18 +453,31 @@ func ExecuteDDCmd(ledName string) {
 }
 
 func doDD(count int) {
```
Comment (rvs) [inline pkg/pillar/cmd/ledmanager/ledmanager.go:455]: Can we change the name of the func to make it mnemonic?

File: pkg/pillar/cmd/ledmanager/ledmanager.go [hunk 2]
```diff
@@ -451,18 +453,31 @@ func ExecuteDDCmd(ledName string) {
...
-			log.Debugln("dd error: ", err)
-		}
+		err = fmt.Errorf("doDD: Failed on open: %s", err)
+		log.Error(err.Error())
 		return
 	}
-	log.Debugf("ddinfo: %s", stdout)
+	defer handler.Close()
+	for i := 0; i < count; i++ {
+		data := make([]byte, bufferLength)                       //4M buffer
```
Comment (rvs) [inline pkg/pillar/cmd/ledmanager/ledmanager.go:466]: Why does this data buffer sit inside of for loop?

**Review by rvs — APPROVED** [review]
Looks reasonable -- but two small nits before we can pull this.

File: pkg/pillar/cmd/ledmanager/ledmanager.go (hunk 1 above)
Comment (eriknordmark) [reply pkg/pillar/cmd/ledmanager/ledmanager.go:455]: Yeah like uncachedDiskRead() or lightDiskLED()

**Review by rvs — APPROVED** [review]
LGTM

**Pull request merged**